webapp/src/Service/ImportExportService.php

```diff
@@ -74,7 +73,7 @@ public function getContestYamlData(Contest $contest): array
                 'label' => $contestProblem->getShortname(),
                 'letter' => $contestProblem->getShortname(),
                 'name' => $contestProblem->getProblem()->getName(),
-                'short-name' => $contestProblem->getProblem()->getExternalid(),
+                'short_name' => $contestProblem->getProblem()->getExternalid(),
```

Comment:
This one doesn't *really* matter since it's not in the spec, but I'm fine changing it.

Reply:
I prefer consistency so I hope we can rename them all somewhere in the future.

We could do those later when we develop DJ9 (not sure if this would be breaking BC), or I can leave both notations and we remove the `short-name` in another version.

Reply:
Changed for id